src/dynimage.rs

```diff
@@ -286,6 +287,11 @@ impl DynamicImage {
         image_to_bytes(self)
     }
 
+    /// Return a view on the raw sample buffer.
+    pub fn as_flattened(&self) -> FlatSamples<&[u8]> {
```

Comment:
How about as_flattened -> as_flat_samples

Reply:
I like `FlatSamples` slightly better, but I don't feel strongly either way

Reply:
It has the association to `image planes`, which are possible here in contrast to `ImageBuffer`, so then it'll stay.